integration/setup/bastion_instance.go

```diff
@@ -578,3 +381,54 @@ func terminateBastionInstance(ctx context.Context, clusterID string, config Conf
 
 	return nil
 }
+
+func generateBastionUserData(ctx context.Context) (string, error) {
+	type IgnitionConfig struct {
+		Config   interface{}
+		Timeouts interface{}
+		Version  string
+	}
+	type UserData struct {
+		Ignition IgnitionConfig
+		Networkd interface{}
+		Passwd   interface{}
+		Storage  interface{}
+		Systemd  interface{}
+	}
+
+	ts := oauth2.StaticTokenSource(
```

Comment:
Check https://github.com/giantswarm/aws-operator/blob/61b2eed3f5f35ca02b420b1bba001bbf260009b0/integration/setup/setup.go#L306-L337. I once started to write some abstraction code for fetching information from the `installations` repo. Let's use this if possible.